src/map/modifier.h

```diff
@@ -1020,7 +1023,7 @@ enum class Mod
     // 221 to 222
     // 274 to 275
     //
-    // SPARE = 1083 and onward
+    // SPARE = 1084 and onward
```

Comment:
I know this is a bitch to ask, but can we use IDs 221 and 222 and be done with that blank range?

Reply:
Sure!